src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1203,7 +1203,7 @@ public class OpcodeCall : Opcode
         protected override string Name { get { return "call"; } }
         public override ByteCode Code { get { return ByteCode.CALL; } }
 
-        public static Type ARG_MARKER_TYPE = typeof(KOSArgMarkerType); // Don't query with reflection at runtime - get the type just once and keep it here.
```

Comment:
there was no protection from having this typed changed at runtime so i made it a property